pages/contact.js

```diff
@@ -43,15 +60,16 @@ export default function Contact() {
         console.log("form submitted successfully !!!");
```

Comment:
Hey Sarah, weldone! Would we be needing the console.log here?

Reply:
thanks! no we can take that out. I think we should probably add some confirmation on the page at some point too but I just didn't get there